sway/commands/workspace.c

```diff
@@ -216,6 +217,9 @@ struct cmd_results *cmd_workspace(int argc, char **argv) {
 			free(name);
 		}
 		workspace_switch(ws, no_auto_back_and_forth);
+		if(config->focus_follows_mouse == FOLLOWS_ALWAYS){
```

Comment:
Be sure to adhere to the style guide. 
- `if(` -> `if (
- `){` -> `) {`

Reply:
whoops, fixed